pkg/storage/kubernetes.go

```diff
@@ -20,11 +21,20 @@ import (
 type kubernetesClient struct {
 	generic
 	client occlient.Client
+
+	// if we don't have access to the local config
+	// we can use the deployment to call ListFromCluster() and
+	// directly list storage from the cluster without the local config
+	deployment *v1.Deployment
 }
 
 // Create creates a pvc from the given Storage
 func (k kubernetesClient) Create(storage Storage) error {
 
+	if k.componentName == "" || k.appName == "" {
+		return fmt.Errorf("the component name, the app name or both are empty")
```

Comment:
Do you think it would help to add more context on why they are required?

Reply:
Added

Reply:
A message indicating how to fix the error would be simpler:

"the component name and app name should be provided"

Reply:
>Do you think it would help to add more context on why they are required?
I meant adding more context in the error message.

>"the component name and app name should be provided"
This indicates that both are required, but we can work with either of them.

Reply:
> This indicates that both are required, but we can work with either of them.

Both of them are required values

> I meant adding more context in the error message.

Updated the error message.